### PR TITLE
Add Ambassador Program 2.0 blog post

### PR DESCRIPTION
**Pull request by travisbeckham**
https://deploy-preview-1881--linkerdio.netlify.app/2024/11/25/ambassador-program/

### CODE REVIEW

**Review by kflynn — APPROVED** [review]
Ship it!

File: linkerd.io/content/blog/2024/1125-ambassador-program/index.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+## What Do Linkerd Ambassadors Do?
+
+Linkerd Ambassadors are the voice of our community, spreading the message of
+simplicity and empowering others to embrace it. Every Ambassador contributes in
+their own way:
+
+- **Educate:** Give talks, write blog posts, create videos, host meetups, and
+  answer questions in the Linkerd Slack.
+- **Foster Collaboration:** Build strong relationships and encourage
+  contributions..
```
Comment (CathPag) [inline linkerd.io/content/blog/2024/1125-ambassador-program/index.md:56]:
```suggestion
  contributions.
```

File: linkerd.io/content/blog/2024/1125-ambassador-program/index.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+- **Grow the Community:** Onboard new users and help them understand the power
+  of Linkerd.
+- **Provide Feedback:** Share community insights and suggestions with the
+  maintainers to keep improving Linkerd.
+- **Promote Globally:** Amplify Linkerd’s message to a wider audience and
```
Comment (CathPag) [inline linkerd.io/content/blog/2024/1125-ambassador-program/index.md:61]:
```suggestion
- **Promote Globally:** Amplify Linkerd’s message to a wider audience.
```

File: linkerd.io/content/blog/2024/1125-ambassador-program/index.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  showcase its benefits.
```
Comment (CathPag) [inline linkerd.io/content/blog/2024/1125-ambassador-program/index.md:62]:
```suggestion

```

**Review by CathPag — COMMENTED** [review]
Two tweaks I didn't see in the Google Doc

**Pull request merged**